cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityOperatorTest.java

```diff
@@ -233,6 +236,9 @@ public void testTemplate() {
         Map<String, String> saLabels = TestUtils.map("l5", "v5", "l6", "v6");
         Map<String, String> saAnnotations = TestUtils.map("a5", "v5", "a6", "v6");
 
+        Map<String, String> rbLabels = TestUtils.map("l9", "v9", "l10", "v10");
+        Map<String, String> rbAnots = TestUtils.map("a9", "v9", "a10", "v10");
```

Comment:
You should use different set of labels and annos for Role and RolBinding. That way, it tests that you do not mix them as well.